src/lib/common/Format.cpp

```diff
@@ -70,6 +71,19 @@ Format stringToFormat(const std::string& s)
   return NOFORMAT;
 }
 
+/* ****************************************************************************
+*
+* isFloat -
+*/
+bool isFloat(const std::string& s)
+{
+  // Thanks to http://stackoverflow.com/questions/447206/c-isfloat-function
+  std::istringstream iss(s);
+  float f;
+  iss >> std::noskipws >> f; // noskipws considers leading whitespace invalid
+  // Check the entire string was consumed and if either failbit or badbit is set
+  return iss.eof() && !iss.fail();
+}
 
```

Comment:
Not one single empty line in this function ...
It is very short, but I still get dizzy watching it ... :-D

The first comment I would move to the function-comment.
After aligned local variables, am empty line, please.

If you want you could describe what the function does in the comment-header and have the function itself free of comments (a matter of taste).